src/App/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
You can remove this file since you're not using this testing library. (doesn't hurt anything to have it, though)